siac/hostcmd.go

```diff
@@ -152,35 +152,50 @@ func hostcmd() {
 		Add(fm.PotentialStorageRevenue).
 		Add(fm.PotentialDownloadBandwidthRevenue).
 		Add(fm.PotentialUploadBandwidthRevenue)
-	fmt.Printf(`Host info:
-	Storage:      %v (%v used)
-	Price:        %v / TB / Month
-	Max Duration: %v Blocks
-
-	Accepting Contracts: %v
-	Anticipated Revenue: %v
-	Locked Collateral:   %v
-	Risked Collateral:   %v
-	Revenue:             %v
-	Lost Revenue:        %v
-	Lost Collateral:     %v
-`, filesizeUnits(int64(totalstorage)), filesizeUnits(int64(totalstorage-storageremaining)),
-		price, is.MaxDuration, accept, currencyUnits(totalPotentialRevenue),
-		currencyUnits(fm.LockedStorageCollateral), currencyUnits(fm.RiskedStorageCollateral),
-		currencyUnits(totalRevenue), currencyUnits(fm.LostRevenue),
-		currencyUnits(fm.LostStorageCollateral))
+	// determine the display method for the net address.
+	netaddr := es.NetAddress
+	if is.NetAddress == "" {
+		netaddr += " (automatically determined)"
+	} else {
+		netaddr += " (manually specified)"
+	}
```

Comment:
At some point I was displaying it in the non-verbose output, but I felt like it would make the future lives of programmers easier if this code were outside.